helper/pluginutils/loader/init.go

```diff
@@ -457,10 +457,11 @@ func (l *PluginLoader) validatePluginConfig(id PluginID, info *pluginInfo) error
 	}
 
 	// Parse the config using the spec
-	val, diag := hclutils.ParseHclInterface(info.config, spec, nil)
+	val, diag, diagErrs := hclutils.ParseHclInterface(info.config, spec, nil)
 	if diag.HasErrors() {
-		multierror.Append(&mErr, diag.Errs()...)
-		return multierror.Prefix(&mErr, "failed parsing config:")
+		multierror.Append(&mErr, diagErrs...)
+		return multierror.Prefix(&mErr, "failed to parse config")
```

Comment:
I'd keep the trailing colon here, as `Prefix()`  prefixes all elements rather than just once on top.  I have a sample output in https://play.golang.org/p/sj0JcGRRVGH .

Though, I'd suggest avoiding `multierror` formatting.  I find it awkward and we can reuse the formatter from other callers.